### ISSUE TITLE
Undefined index: #primary in...

### ISSUE THREAD

**Issue by ga12** [SU-SWS/open_framework#129]
I have installed Open Framework Subtheme 1.0 in a test site and began getting the following:

Notice: Undefined index: #primary in open_framework_preprocess_page() (line 72 of /home/content/89/10801789/html/dru721/sites/all/themes/open_framework-7.x-2.04/template.php).

Could someone share with me how I can resolve this?

Thank you.
ga12@cornell.edu

**Comment by meganem** [comment]
Thanks for submitting the issue. We will see if we can replicate.

**Comment by whatnickcodes** [comment]
Hmmm. I'm also getting this issue. Any updates?

**Comment by whatnickcodes** [comment]
I added this (above line 72 inside the function where this occurs) to handle for the error.

``` php
// Checks if tabs are set
if (!isset($vars['tabs']['#primary'])) $vars['tabs']['#primary'] = FALSE;

// Replace tabs with drop down version
$vars['tabs']['#primary'] = _bootstrap_local_tasks($vars['tabs']['#primary']);
```

Seems to resolve the problem. There's probably a better fix. Will look into later. Let me know your thoughts guys.

Thanks,

Nick

Edit: If someone wants to double check this, I'll go ahead and submit a pull request.
Edit 2: I also found, ensuring that the Open Framework base Modules are installed will prevent this error from occuring.
